src/app/page/Request/foodbox/FoodboxFlow.jsx

```diff
@@ -1,23 +1,15 @@
 import { Paper, Tab, Tabs } from "@material-ui/core";
 import CircularProgress from "@material-ui/core/CircularProgress";
 import { makeStyles, withStyles } from "@material-ui/core/styles";
-import React, { useState } from "react";
+import React, { useReducer } from "react";
```

Comment:
Haven't had time to read through useReducer docs yet, can't comment too much on implementation around this.